main.cpp

```diff
@@ -93,18 +93,18 @@ toe.wiggleUpandDown();
 toe.growToenail();
 //  6)
 //  Noun:     hand
-//  action 1: the hand waves
-hand.wave();
-//  action 2: the hand gives a thumbs up
+//  action 1: the hand twists side to side
+hand.twistsSidetoSide();
+//  action 2: the hand forms a thumbs up
 hand.thumbsUp();
-//  action 3: the hand gives a thumbs down
+//  action 3: the hand forms a thumbs down
 hand.thumbsDown();

```

Comment:
The pseudocode is missing the action word `form`
```
hand.formThumbsUp();
hand.formThumbsDown();
```
Remember: Name things what they are or what they do.  
It will make your code easier to understand for anyone else who looks at it, including yourself months from now.

This habit (Name things what they are or what they do) needs to be established very early on in your coding journey.  If you haven't been following this guideline, now is the time to start.
- [ ] revise